### PR TITLE
Add Chromecast support for device config

### PR DESCRIPTION
**Pull request by sagaalicekarlsson**
Modifies the functionality of selecting Chromecast devices to not require a selected playlist, allowing for selecting Chromecast devices in the configuration, starting the cast first when a playlist has been selected as well. This allows for playing playlists on your Chromecast with only one click (with of course a few seconds required the first time to set up the cast)!

### CODE REVIEW

**Comment by sagaalicekarlsson** [comment]
Please tell me if there is any questions about the PR and if there is something you would like to be changed, and thank you for a very appreciated card! :)

**Comment by fondberg** [comment]
Thanks for your PR. I am right now fixing some things. Can you merge master when I'm done?

**Comment by fondberg** [comment]
Now it should be done

File: src/SpotifyCard.js [hunk 1]
```diff
@@ -231,6 +264,10 @@ export default class SpotifyCard extends Component {
     return playlist.id === playingPlaylistId ? 'playing' : '';
   }
 
+  isCastType(type) {
+    return type == 'CastAudio' || type == 'CastVideo' || type == 'CastPlaceholder';
```
Comment (fondberg) [inline src/SpotifyCard.js:268]: Isn't CastGroup also one?

**Comment by fondberg** [comment]
Otherwise it looks good!

File: src/SpotifyCard.js (hunk 1 above)
Comment (sagaalicekarlsson) [reply src/SpotifyCard.js:268]: These should be the ones, right? https://developer.spotify.com/documentation/web-api/reference/player/get-a-users-available-devices/#device-types

If so, these two (and the placeholder one) should be the only ones.

Comment (fondberg) [reply src/SpotifyCard.js:268]: These are home assistant cast types.

**Comment by fondberg** [comment]
I've been busy I  the new version. If it is on with you I'll add your feature to that one

**Comment by sagaalicekarlsson** [comment]
Hi,

Absolutely! I'm away and have not been able to add the missing cast type.
But feel free to add it if you wish.

/Saga

**Comment by fondberg** [comment]
This functionality is included in v2. v1 is not supported anymore

**Pull request closed without merging**